Python/chapter01/1.9 - String Rotation/camilo_solution_1.9.py

```diff
@@ -0,0 +1,27 @@
+"""
+String Rotation:Assume you have a method isSubstring which checks if one word is a
+substring of another. Given two strings, sl and s2, write code to check if s2 is
+a rotation of sl using only one call to isSubstring (e.g.,"waterbottle" is a
+rotation of"erbottlewat").
+"""
+
+def isSubstring(s1,s2):
+    if s2 in s1: #check to see if the seqence of s2 is found in s1
```

Comment:
```python
return s2 in s1
```

suffices and you avoid the most common [antipattern](https://ucsb-cs16.github.io/topics/antipatterns/).